vtr_flow/tasks/regression_tests/vtr_reg_weekly/koios_bwave_large/config/config.txt

```diff
@@ -35,4 +36,4 @@ qor_parse_file=qor_standard.txt
 pass_requirements_file=pass_requirements.txt
 
 #Script parameters
-script_params=-track_memory_usage -crit_path_router_iterations 100 --route_chan_width 300 --target_utilization 0.28
\ No newline at end of file
+script_params=-track_memory_usage -crit_path_router_iterations 100 --route_chan_width 400 --target_utilization 0.12
```

Comment:
What's the reason for the .12 target_utilization?  We should at least comment it (and it is quite a bit lower than the old .28).